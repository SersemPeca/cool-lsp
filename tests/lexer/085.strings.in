"too long --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- 1025 chars"
