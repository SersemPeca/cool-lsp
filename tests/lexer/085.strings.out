#1 ERROR: String constant too long
